### DISCUSSION TITLE
Privacy implications of each instance (network provider)?

### DISCUSSION THREAD

**Discussion by Morasithil** [searxng/searx-instances#420] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

Poor response in discussions

**Describe the solution you'd like**

Understanding country and network impact on instance privacy

**Describe alternatives you've considered**

**Additional context**

If i understand correctly, whoever is hosting a searxng instance has the capability to log all search-requests including the useragent and ip-adress.
From what i've read anyone can host a searxng instance.
That means google, facbook, the CIA, really anyone could host an instance and keep logs of all searches.
So it would be wise to investigate which instance appears to most trustworthy as i don't have the technical knownledge to host my own instance.
So i went here https://searx.space/ and figured there are two major factors to pick an instance, country and network.
Where network stands for hosting provider?
Why i think country and provider are the most important factors is because some countrys (for example switzerland) have stronger privacy laws than others.
As for hosting providers that's a little more research as i had to look-up every single one of them to make up my mind.
To determine wheather a hosting provider is trustworthy i visited their official website and did a scan of the outbound connections that site has, so when a hosting providers website is connected to google-analytics this is a clear red flag for me and i won't use any instance that is hosted by this provider.
I struggle to understand the concept of a server being hosted in for example finland but the network (hosting provider) of that instance is located in germany, what do i make of that? What impact does this have on privacy?
Does the country where the server is physically located have the options to log all traffic or the network provider or both?

Here's a specific question about this instance https://search.projectsegfau.lt/
Country is luxembourg and network is PONYNET.
When i searched for PONYNET (using bravesearch) this was the result: https://search.brave.com/search?q=PONYNET
In short, various websites claim that PONYNET has been known to be a malware distribution center, used for spam, cybercrime and mass phishing campaigns.
Now my impression is i should not use an instance with this network provider, unless anyone here changes my mind by explaining things.

There are a few instances i was interested in but failed to find any information on their network.
https://searxng.ch/ network = NINE
https://searx.fmac.xyz/ network = THE-1984-AS
How do i find information about these networks? Atleast bravesearch doesn't seem to be capable of finding anything useful.

I've also noticed that some instances that where previously hosted by a swiss network (CH) now use (IS) iceland instead or (US) america and so on, when i chose an instance purely because of it's network provider and then after some months realize that the network provider has changed... this completely defeats the purpose of using that instance.
So i would have to manualy check the searxng instances page from time to time to make sure the network provider hasn't change?!

**Comment by unixfox** [comment, marked as answer]
Let's continue the discussion in https://github.com/searxng/searxng/discussions/2926
